pretix_mollie/migrations/0001_initial.py

```diff
@@ -0,0 +1,26 @@
+# Generated by Django 4.2.15 on 2025-01-31 12:42
+
+from django.db import migrations
+from django_scopes import scopes_disabled
+
+from pretix.base.models import OrderPayment, OrderRefund
+
+
+@scopes_disabled()
+def migrate_applepay(apps, schema_editor):
+    OrderPayment.objects.filter(provider="mollie_applepay").update(
+        provider="mollie_creditcard",
+        migrated=True,
+    )
+
+    OrderRefund.objects.filter(provider="mollie_applepay").update(
+        provider="mollie_creditcard",
+    )
+
+class Migration(migrations.Migration):
+
+    dependencies = []
+
+    operations = [
+        migrations.RunPython(migrate_applepay, migrations.RunPython.noop)
```

Comment:
FYI I checked that this affects <10k rows on pretix Hosted, otherwise we might need to do it differently for to avoid a performance issue during the migration